src/Components/Organisms/Main-Graphic/MainGraphic.tsx

```diff
@@ -0,0 +1,41 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { Grid, Typography } from '@mui/material';
+import { height } from '@mui/system';
+
+import BannerLG from '../Main-Graphic/bannerLG.png';
+
+function MainGraphic() {
+  return (
+    <Box>
+      <Grid container height={'100%'}>
+        <Grid
+          item
+          xs={12}
+          md={7}
+          sx={{
```

Comment:
Have a look at the smaller screens: 
![image](https://user-images.githubusercontent.com/58532638/163544876-0ea27856-68e8-4f5f-a505-13d0dc594c45.png)